pages/common/sops.md

```diff
@@ -0,0 +1,29 @@
+# sops
+
+> SOPS: Secrets OPerationS.
+> Simple and flexible tool for managing secrets.
+> More information: <https://github.com/mozilla/sops>.
+
+- Encrypt a file:
+
+`sops -e {{path/to/myfile.json}} > {{path/to/myfile.enc.json}}`
+
+- Decrypt a file in `cat` fashion using the `-d` flag:
```

Comment:
```suggestion
Decrypt a file to the standard output:
```
credits: @sbrl

Reply:
You missed the `-`, so I added it for you :-)